File: python/sprint1_nonfinals/l.py
from typing import Tuple

def get_excessive_letter(shorter: str, longer: str) -> str:
    # Здесь реализация вашего решения
    pass

def read_input() -> Tuple[str, str]:
    shorter = input().strip()
    longer = input().strip()
    return shorter, longer

shorter, longer = read_input()
print(get_excessive_letter(shorter, longer))

File: cpp/sprint1_nonfinals/l.cpp
#include <iostream>
#include <string>

using namespace std;

char GetExcessiveLetter(const string &shorter, const string &longer) {
	// Здесь реализация вашего решения

}

void print(char result) {
	cout << result << endl;
}

int main() {
	string shorter, longer;
	cin >> shorter >> longer;
	print(GetExcessiveLetter(shorter, longer));
}
